dbt_invoke/properties.py

```diff
@@ -258,27 +258,37 @@ def _transform_ls_results(ctx, **kwargs):
     """
     # Run dbt ls to retrieve resource path and json information
     _LOGGER.info('Searching for matching resources...')
-    result_lines_path = _utils.dbt_ls(
-        ctx,
-        supported_resource_types=_SUPPORTED_RESOURCE_TYPES,
-        logger=_LOGGER,
-        **kwargs,
-    )
-    result_lines_dict = _utils.dbt_ls(
+    potential_results = _utils.dbt_ls(
         ctx,
         supported_resource_types=_SUPPORTED_RESOURCE_TYPES,
         logger=_LOGGER,
         output='json',
         **kwargs,
     )
-    results = dict(zip(result_lines_path, result_lines_dict))
-    # Filter dictionary for existing files and supported resource types
-    results = {
-        k: v
-        for k, v in results.items()
-        if v['resource_type'] in _SUPPORTED_RESOURCE_TYPES
-        and Path(ctx.config['project_path'], k).exists()
-    }
+    potential_result_paths = None
+    results = dict()
+    for i, potential_result in enumerate(potential_results):
+        if 'original_file_path' in potential_result:
+            potential_result_path = potential_result['original_file_path']
+        # Before dbt version 0.20.0, original_file_path was not
+        # included in the json response of "dbt ls". For older
+        # versions of dbt, we need to run "dbt ls" with the
+        # "--output path" argument in order to retrieve paths
+        else:
+            if potential_result_paths is None:
```

Comment:
Doesn't seem to make a difference, but was just curious why `if potential_result_paths is None` instead of `if not potential_result_paths`? (Feel free to resolve this after answering)

Reply:
You're right, it is not really necessary at this point, but I was just thinking ahead about if we ever need/want to separately handle the case where `potential_result_paths` returns an empty list (which is "falsy").